.github/workflows/tests_lkcpu_python.yml

```
@@ -48,6 +48,7 @@ jobs:
     uses: ./.github/workflows/build_and_cache_Kokkos_linux.yml
     with:
       os: ubuntu-22.04
+      kokkos_version: ${{ inputs.lightning-version == 'stable' && '4.2.00' || '4.3.01' }} #To be updated next Release.
```

Note that with the recent changes, 4.2 will likely fail if hit, as new functionality was added in 4.3 `e.g. kokkos_swap`. Though, correct me if wrong.

---

4.2.00 will only be used if calling the stable version. Currently, the stable version is based on the 4.2.00 version.

---

Ah, got it. :+1: 

Should this become 
```suggestion
      kokkos_version: ${{ (inputs.lightning-version == 'stable' && '4.2.00') || '4.3.01' }} #To be updated next Release.
```
for clarity then? Or will that influence the parsing?

---

I don't remember if this will break something or not. GitHub is kind of nitty pick with how we code workflows.
I'm following here their [prescription](https://docs.github.com/en/actions/learn-github-actions/expressions#example) for a ternary operator.

---

Ok, I'll abide by that.